health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/WorkflowCommentPopUp.js

```
@@ -0,0 +1,174 @@
+
+        // Ensure the censusList is an array
+        if (!Array.isArray(censusList)) {
+            throw new Error("Census is not an array");
+        }
+
+        // Map through each item in the censusList
+        return {
+            ...payloadObject,
+            Census: censusList.map(item => {
+                // Create a shallow copy of the current item
+                let updatedItem = { ...item };
```

Biome

[error] This let declares a variable that is only assigned once.

'updatedItem' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)